src/Admin/AdminConsole/Views/Shared/_OrganizationForm.cshtml

```diff
@@ -70,9 +70,10 @@
                     @{
                         var planTypes = Enum.GetValues<PlanType>()
                             .Where(p =>
-                                Model.Provider == null ||
-                                (Model.Provider != null
```

Comment:
Removed redundant `Model.Provider != null` since it's always not `null` if it reaches this side of the `||`.